### DISCUSSION TITLE
Copy and Past not working?

### DISCUSSION THREAD

**Discussion by Tobiaswec** [jlesage/docker-jdownloader-2#192] [Q&A]
Is there something wrong with my config or is copy and paste not working in the web GUI?

**Comment by me1299** [comment]
I have the exact same problem. Just installed, seems to be running just fine but the clipboard seems to be isolated in the webpage. When I type something inside the GUI and copy then that clipboard segment is saved. But it's not available outside the website.

**Comment by mihawk90** [comment]
JD (or rather the remote session) can only interact with your browser, not your system clipboard. Whatever you copy inside JD gets saved to the Clipboard panel on the left, where you can copy it to your system clipboard.
![image](https://github.com/user-attachments/assets/259a196c-4c77-4397-b571-963e6c3a7526) ![image](https://github.com/user-attachments/assets/b3a5ff17-0967-461d-8920-c6c4d8c58e39)
